src/libstd/path.rs

```diff
@@ -185,6 +187,9 @@ impl<'a> Prefix<'a> {
             os_str_as_u8_slice(s).len()
         }
         match *self {
+            #[cfg(target_os = "redox")]
+            Verbatim(x) => 1 + os_str_len(x),
```

Comment:
Due to having only an extra b':', we add 1 to the verbatim string length instead of 4